prime-router/metadata/hl7_mapping/ORU_R01/base/order-observation.yml

```diff
@@ -30,8 +30,10 @@ elements:
     schema: timing-quantity
 
     # Appending AOE questions at the end of all observations
-  - name: observation-result
-    resource: '%resource.result.resolve() | %service.supportingInfo.resolve()'
+  - name: observation-result-with-aoe
+    #TODO: Enable this once we can suppress AOE questions in sender transforms
+#    resource: '%resource.result.resolve() | %service.supportingInfo.resolve()'
+    resource: '%resource.result.resolve()'
     schema: observation-result
```

Comment:
This needs to be turned off for now until we find a way to suppress AOEs. A ticket was created [here](https://app.zenhub.com/workspaces/platform-6182b02547c1130010f459db/issues/gh/cdcgov/prime-reportstream/8693).